lib/tanker/identity.rb

```diff
@@ -117,9 +117,25 @@ def self.get_public_identity(serialized_identity)
       public_identity = {}
       public_keys.each { |key| public_identity[key] = identity.fetch(key) }
 
+      if identity['target'] == 'email'
+        identity['target'] = 'hashed_email'
+        public_identity['value'] = Base64.strict_encode64(Crypto.generichash(public_identity['value'], BLOCK_HASH_SIZE))
```

Comment:
IMO the target should always be set to "hashed_email". Otherwise we can't known if a public identity is an old `email` with or without hash

Reply:
This is what this does, I think =)  
If the private prov ident had target `email`, get_public_identity updates it to a `hashed_email`

Reply:
... Or do you mean the private identity should be `hashed_email` too? But then don't they have to do a migration of the private identities?